src/main/java/ui/json/Global.java

```diff
@@ -0,0 +1,148 @@
+/* Licensed under MIT 2024. */
+package ui.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+/**
+ * Custom Serializer for Global to provide additional functionality that is
+ * ignored when writing to the JSON file.
+ */
+@JsonSerialize(using = GlobalSerialzer.class)
+public class Global {
```

Comment:
I still don't get it. You could use `@JsonIgnore` if you want to ignore things while saving.

Reply:
Didn't know that, first time working with it. Tested and fixed